test.sh

```diff
@@ -27,7 +27,16 @@ start eslint
 node_modules/eslint/bin/eslint.js .
 end eslint
 
+if [[ "$CI" != 'true' ]]
```

Comment:
Add an reminder here, that this is covered by pre-commit.

Reply:
Sure.